### ISSUE TITLE
clang-format: break added to macro define with ColumnLimit: 0

### ISSUE THREAD

**Issue by 2a56267d-9ebe-4d68-8c73-88d5a36c7dc4** [llvm/llvm-project#49164]
|  |  |
| --- | --- |
| Bugzilla Link | [49820](https://llvm.org/bz49820) |
| Version | 11.0 |
| OS | Windows NT |

## Extended Description 
Formatting following code with 'ColumnLimit: 0'
```
#define STRINGIFY(t) #t
#define MAKEVERSIONSTRING(x, y, z, build) STRINGIFY(x) "." STRINGIFY(y) "." STRINGIFY(z) "." STRINGIFY(build)
```

Results in:
```
#define STRINGIFY(t) #t
#define MAKEVERSIONSTRING(x, y, z, build) STRINGIFY(x) \
"." STRINGIFY(y) "." STRINGIFY(z) "." STRINGIFY(build)
```

No break should be added as the column limit is 0 (unlimited).
If the column limit is set to e.g. 120, the break is not added.

**Comment by mkurdej** [comment]
Review: https://reviews.llvm.org/D116859.

**Comment by mkurdej** [comment]
Fixed in 47a9eb2.
